### DISCUSSION TITLE
Feedback on plugin compatibility check

### DISCUSSION THREAD

**Discussion by tolzhabayev** [grafana/grafana#53898] [Plugins]
Hey there 👋 
we would love to hear your feedback on the `is-compatible` function.

https://github.com/grafana/grafana-plugin-examples/#api-compatibility

This tool is in very much beta phase so subject to change and improve a lot over the next iterations

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
